libminifi/test/flow-tests/FlowControllerTests.cpp

```diff
@@ -193,7 +198,7 @@ TEST_CASE("Extend the waiting period during shutdown", "[TestFlow4]") {
   auto controller = testController.controller_;
   auto root = testController.root_;
 
-  int timeout_ms = 1000;
+  unsigned int timeout_ms = 1000;
```

Comment:
As this is a duration its type could also be std::chrono::duration, with no _ms suffix needed then.